### DISCUSSION TITLE
Buying licence for company

### DISCUSSION THREAD

**Discussion by Thomastk67** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1156] [General]
Hallo,
Id like to buy licence for my Company. I'd like to get a bill, and my company residence is in Europe. We have valid UE number.
How can I do it?

**Comment by MatthiasKerbl** [comment]
Hi @Thomastk67,

Thank you for your interest in AlDente. Please send me your request to the following email address  and I will come back to you with more details.

Best,
Matthias
